docs/source/AdministratorGuide/Systems/WorkloadManagement/Pilots/Pilots3.rst

```diff
@@ -28,15 +28,9 @@ The pilot.json file is always kept in sync with the content of the Configuration
 At every configuration update, the pilot.json file content will also be updated.
 
 
-If you use the DIRAC webserver please
-
-- add the following option to the WebApp CS section::
-       
-    /WebApp/StaticDirs=pilot
-       
-- create the following directory in the DIRAC webserver machine::
-   
-    mkdir /opt/dirac/webRoot/www/pilot/
+Starting with web portal version v4r2, the file uploads is completely on the balancer side.
```

Comment:
I would put the DIRAC version (v7r2) rather than the WebApp version, which is "hidden" inside the DIRAC version.

Reply:
fixed